nautobot/extras/templatetags/url_encode.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+from urllib.parse import urlencode
+
+register = template.Library()
+
+
+@register.simple_tag
+def url_encode(val=None):
+    return urlencode(val)
```

Comment:
Extras, utilities, core... Oh my!

We have a handful of tags already and since this isn't particular around any extra app, can you move this to here: https://github.com/nautobot/nautobot/blob/develop/nautobot/utilities/templatetags/helpers.py

Reply:
soz, unfamiliar with some of the wider project layout. fixed